src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue

```diff
@@ -369,6 +379,18 @@
     const MAXIMUM_ALLOWED = 5
     return criteria.value.length >= MAXIMUM_ALLOWED
   })
+
+  watch(
+    () => props.errors,
+    () => {
```

Comment:
Se for possível, minha sugestão seria isolar isso em uma função e comentar/documentar ela, para que fique mais claro o propósito desse Watch.